tests/v0_tests/test_get_settings.py

```diff
@@ -58,6 +60,7 @@ def test_custom_settings(self):
 
         self.assertIn('index_defaults', index_settings)
         self.assertIn('number_of_shards', index_settings)
+        self.assertIn('number_of_shards', index_settings)
```

Comment:
is this meant to be "number_of_replicas"?

Reply:
updated